### DISCUSSION TITLE
请问移位寄存器能否用Nexperia的代替？

### DISCUSSION THREAD

**Discussion by cyijun** [peng-zhihui/HelloWord-Keyboard#72] [Q&A] [answered]
不是很懂数电，读了数据手册看起来定义都是一样的，请问在这个应用里是不是只有封装上的区别？

手册地址：
https://assets.nexperia.com/documents/data-sheet/74HC_HCT165.pdf

**Comment by xingrz** [comment, marked as answer]
只要是74HC165就可以，甚至淘宝上5毛钱一颗的都可以

**Reply by cyijun** [reply]
谢谢
